src/main/java/frc/robot/Robot.java

```diff
@@ -78,13 +87,17 @@ public void robotInit() {
      *
      * <p>This runs after the mode specific periodic functions, but before LiveWindow and
      * SmartDashboard integrated updating.
+     *
+     * You thought this was a comment that would explain about this function, but it was me, DIO!
      */
     @Override
     public void robotPeriodic() {
         // Runs the Scheduler.  This is responsible for polling buttons, adding newly-scheduled
         // commands, running already-scheduled commands, removing finished or interrupted commands,
         // and running subsystem periodic() methods.  This must be called from the robot's periodic
         // block in order for anything in the Command-based framework to work.
+
+        System.out.println(RobotContainer.colorSensor.getReading());
```

Comment:
Add a todo to remember to delete this. printlns kill performance

Reply:
Did that